common/src/main/scala/gu/msnotifications/package.scala

```diff
@@ -1,24 +1,24 @@
 package gu
 
-import java.util.Base64
-
-import models.{TopicType, Topic, UserId}
+import models.{Topic, UserId}
 
 package object msnotifications {
 
-  private def encode(string: String): String =
-    Base64.getUrlEncoder.encodeToString(string.getBytes("UTF-8"))
+  private def encodeBase16(string: String): String =
+    string.getBytes("UTF-8").map("%02X".format(_)).mkString.toLowerCase
 
-  private def decode(string: String): String =
-    new String(Base64.getUrlDecoder.decode(string), "UTF-8")
+  private def decodeBase16(string: String): String = {
+    val bytes = string.grouped(2).map(unit => java.lang.Byte.parseByte(unit, 16))
```

Comment:
Is there another name that you could use instead of unit to avoid confusion with Unit